### PR TITLE
Export OSD cluster map flags

### PR DESCRIPTION
**Pull request by tbregolin**
I would like to export the OSD cluster map flags as metrics for monitoring and alerting.

### CODE REVIEW

**Comment by tbregolin** [comment]
@neurodrone Any thoughts on the above?

File: collectors/health.go [hunk 1]
```diff
@@ -152,6 +152,21 @@ type ClusterHealthCollector struct {
 	// This includes object replicas in its count.
 	MisplacedObjectsCount prometheus.Gauge
 
+	// OSDMapFlags
+	OSDMapFlagFull prometheus.Gauge
```
Comment (neurodrone) [inline collectors/health.go]: Would it be possible to run `gofmt` on this file? That should fix this indentation. Other necessary options are covered here: https://github.com/digitalocean/ceph_exporter/blob/luminous/CONTRIBUTING.md.

**Comment by tbregolin** [comment]
@neurodrone Thank you for the review. I've pushed the changes. Sorry for the formatting glitch.

**Comment by tbregolin** [comment]
@neurodrone please let me know if anything else is missing so that we can merge it. Thank you.

**Comment by neurodrone** [comment]
Looks great! Thanks for the changes! (sorry about the delay)

**Pull request merged**